pages/user/[username].tsx

```diff
@@ -0,0 +1,96 @@
+import Meta from "../../components/common/Meta";
+import Post from "../../components/common/Post";
+import { DBPost } from "../../lib/types/PostTypes";
+import { AuthContext } from "../../lib/context/AuthContext";
+import { useContext, useEffect, useState } from "react";
+import useRedirect from "../../lib/hooks/useRedirect";
+import Counter from "../../components/profile/Counter";
+import UpdateForm from "../../components/profile/UpdateForm";
+
+function User({ username, data }: { username: string; data: any }) {
```

Comment:
Any...

Reply:
I need to check this one because here data can be 2 different things if I'm not mistaken

Reply:
In that case you can use a combined type.

Reply:
fixed, but when the real data will be fetched we'll need other types